### PR TITLE
SG-11207 Update Pipeline Tutorial

### PR DESCRIPTION
**Pull request by willis102**
Update the pipeline tutorial to reflect changes in the wizard since it was written, link to more external documentation, and improve clarity.

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/ca68459214528cc04dfca2bd73484eedfebe9daa/index.html)

File: docs/en/toolkit/learning-resources/tutorial.md [hunk 1]
```diff
@@ -93,15 +93,27 @@ Use Shotgun Desktop (Desktop) to take over your project's configuration. RMB cli
...
+
+You can view and edit the storages for your shotgun site in your **Site Preferences**, under the **File Management** section.  You can learn more about these settings [here](https://support.shotgunsoftware.com/hc/en-us/articles/219030938).
+
+Now that you have a storage location selected, you'll choose the name of the directory in that location for your new project.
+
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_06.png" caption="Enter the name of the folder where your project's files will live." %}
+
+For this tutorial, we'll be using a centralized configuration.  The **Distributed Setup** option provides an alternate option that can provide a different set of benefits, and may be the preferred option for studios without fast shared storage.  You can learn more about the pro and cons of different configuration setups in the [Toolkit Administering](https://www.youtube.com/watch?v=7qZfy7KXXX0&list=PLEOzU2tEw33r4yfX7_WD7anyKrsDpQY2d&index=2) presentation.
```
Comment (tannazie) [inline docs/en/toolkit/learning-resources/tutorial.md]: "Toolkit Administration" is the name of the presentation, not "Toolkit Administering".

File: docs/en/toolkit/learning-resources/tutorial.md [hunk 2]
```diff
@@ -826,8 +844,10 @@ For more information on reviewing media in Shotgun, see the [official documentat
...
 
-If there are features or workflows that you feel are outside of the default integrations, then you can always write your own apps. [Here is an excellent document](https://support.shotgunsoftware.com/entries/95440137) to help you get started writing your first app.
+To learn more about developing your own apps, check out the [App Development Guide](https://support.shotgunsoftware.com/hc/en-us/articles/219033158-Developing-Apps).
 
-As always, if you have additional questions about this tutorial or about Shotgun or the Toolkit platform in general, feel free to [submit a ticket](https://support.shotgunsoftware.com/hc/en-us/requests/new).
+Ask questions and learn how other studios are using Toolkit over at the [shotgun-dev Google Group](https://groups.google.com/a/shotgunsoftware.com/forum/#!forum/shotgun-dev).
 
+If there are features or workflows that you feel are outside of the default integrations, then you can always write your own apps. [Here is an excellent document](https://support.shotgunsoftware.com/entries/95440137) to help you get started writing your first app.
```
Comment (tannazie) [inline docs/en/toolkit/learning-resources/tutorial.md]: I think this is reduntant with the first bullet point, no? Points to the same doc (despite the URLs being different).

File: docs/en/toolkit/learning-resources/tutorial.md [hunk 3]
```diff
@@ -826,8 +844,10 @@ For more information on reviewing media in Shotgun, see the [official documentat
 
 Congratulations, you're done! Hopefully this tutorial has given you a starting point for building your own custom pipeline using the Shotgun integrations. You should have an understanding of how to extend the default integrations to meet the specific needs of your studio.
 
-If there are features or workflows that you feel are outside of the default integrations, then you can always write your own apps. [Here is an excellent document](https://support.shotgunsoftware.com/entries/95440137) to help you get started writing your first app.
+To learn more about developing your own apps, check out the [App Development Guide](https://support.shotgunsoftware.com/hc/en-us/articles/219033158-Developing-Apps).
 
-As always, if you have additional questions about this tutorial or about Shotgun or the Toolkit platform in general, feel free to [submit a ticket](https://support.shotgunsoftware.com/hc/en-us/requests/new).
+Ask questions and learn how other studios are using Toolkit over at the [shotgun-dev Google Group](https://groups.google.com/a/shotgunsoftware.com/forum/#!forum/shotgun-dev).
```
Comment (robblau) [inline docs/en/toolkit/learning-resources/tutorial.md]: Should suggest that people subscribe if they aren't already.

File: docs/en/toolkit/learning-resources/tutorial.md (hunk 1 above)
Comment (willis102) [reply docs/en/toolkit/learning-resources/tutorial.md]: 🤦‍♂️ ✅

File: docs/en/toolkit/learning-resources/tutorial.md (hunk 2 above)
Comment (willis102) [reply docs/en/toolkit/learning-resources/tutorial.md]: yep, removed.

File: docs/en/toolkit/learning-resources/tutorial.md [hunk 4]
```diff
@@ -93,15 +93,27 @@ Use Shotgun Desktop (Desktop) to take over your project's configuration. RMB cli
...
+
+You can view and edit the storages for your shotgun site in your **Site Preferences**, under the **File Management** section.  You can learn more about these settings [here](https://support.shotgunsoftware.com/hc/en-us/articles/219030938).
+
+Now that you have a storage location selected, you'll choose the name of the directory in that location for your new project.
+
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_06.png" caption="Enter the name of the folder where your project's files will live." %}
+
+For this tutorial, we'll be using a centralized configuration.  The **Distributed Setup** option provides an alternate option that can provide a different set of benefits, and may be the preferred option for studios without fast shared storage.  You can learn more about the pro and cons of different configuration setups in the [Toolkit Administering](https://www.youtube.com/watch?v=7qZfy7KXXX0&list=PLEOzU2tEw33r4yfX7_WD7anyKrsDpQY2d&index=2) presentation.
+
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_07.png" caption="Make a note of the configuration path you select for the current operating system." %}
```
[the remark this answers is not in the document]
Comment (willis102) [reply docs/en/toolkit/learning-resources/tutorial.md]: I added a sentence that will hopefully make that clearer.  I had already added some info in the captions for the creating storages steps, but I think you're right that the config should get a quick blurb as well.

File: docs/en/toolkit/learning-resources/tutorial.md [hunk 5]
```diff
@@ -276,6 +290,8 @@ For the purposes of this simple pipeline, you will customize the Publisher app t
 
 {% include info title="Note" content="The customization you'll be adding is, admittedly, very simple and fragile. A more robust solution might take into account alternate representations of a surfaced character as well as the asset management side of using external images as texture maps. This example presents only a starting point for building a real-world solution." %}
 
+To learn more about the Publisher and the changes you'll be making, have a look at the [API Documentation](https://developer.shotgunsoftware.com/tk-multi-publish2/).
```
[the remark this answers is not in the document]
Comment (willis102) [reply docs/en/toolkit/learning-resources/tutorial.md]: ✅

File: docs/en/toolkit/learning-resources/tutorial.md (hunk 3 above)
Comment (willis102) [reply docs/en/toolkit/learning-resources/tutorial.md]: ✅

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/ed6e4441e6cdbeb6607e4c91d6e1c01b62d1840e/index.html)

**Pull request merged**